packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SubjectGroupViewer/SubjectGroupViewer.spec.js

```diff
@@ -75,7 +75,7 @@ describe('Component > SubjectGroupViewer', function () {
   })
   
   it('should have a viewBox proportional to width and height', function () {
-    const viewBox = wrapper.find('svg').prop('viewBox')
+    const viewBox = wrapper.find('styledSvg').prop('viewBox')  // The styled <svg> needs a custom .displayName
```

Comment:
Import the exported styled SVG component and you can do something like `wrapper.find(SVG)` instead.